addOns/retire/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- retire: reuse Repo for performance improvement.
```

Comment:
> Performance improvements (Issue 8659).

If you use an amending commit and then force push you won't have to re-squash.

Reply:
Yeah, I am aware. I forgot to fetch upstream changes and did the squash then I fetched the upstream changes and squashed it again that's why I did squash two times.

Reply:
No problem

Reply:
Should still be updated to use the quoted text